### PR TITLE
[postcss-flexbugs-fixes] Updates types for postcss-flexbugs-fixes 5.0

### PR DESCRIPTION
**Pull request by stof**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: version 5 of the plugin is about migrating to postcss 8 instead of postcss 7
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/postcss-flexbugs-fixes/tsconfig.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6"
+            "es6",
+            "es2018"
```
Comment (peterblazejewicz) [inline types/postcss-flexbugs-fixes/tsconfig.json:6]:
```suggestion
            "es2018.Promise"
```

Comment (stof) [reply types/postcss-flexbugs-fixes/tsconfig.json:6]: fixed

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@stof thanks!

**Comment by stof** [comment]
Ready to merge

**Pull request merged**